server/lib/pbench/__init__.py

```diff
@@ -251,6 +251,11 @@ def __init__(self, cfg_name):
                 except (NoOptionError) as exc:
                     raise BadConfig(str(exc))
 
+        try:
+            self.logging_level = self.conf.get("logging", "logging_level")
+        except (NoOptionError, NoSectionError):
+            self.logging_level = "DEBUG"
```

Comment:
If we don't set the level at all, then maybe it should be `INFO`, rather than `DEBUG`.

Reply:
ok